webextension/popup.css

```diff
@@ -130,12 +129,12 @@ form img {
 
 #issueRemoveScreenshot::before {
   content: "__MSG_buttonRemoveScreenshot__";
-  background: url(icons/webcompat.svg) no-repeat;
+  background: url(icons/broken_page.svg) no-repeat;
   padding-left: 20px;
 }
 #issueTakeScreenshot::before {
   content: "__MSG_buttonTakeScreenshot__";
-  background: url(icons/webcompat.svg) no-repeat;
+  background: url(icons/broken_page.svg) no-repeat;
```

Comment:
We will possibly want to change this to the screenshots icon once we get final UX mocks, but it's fine as is.